src/stylesheets/_variables.scss

```diff
@@ -32,4 +32,11 @@
   --malibu__gradient: #8a9dda;
   --new-blue: rgba(138, 157, 218, 0.6);
   --heliotrope: #a687ff;
+
+  /* team page */
+  --dark-mahogany: #32060c;
+  --midnight-blue: #1a1542;
+  --light-purple: #a687ff;
+  --white-blue: #e1edff;
+  --light-aqua: rgba(211, 221, 255, 0.9);
```

Comment:
This is fine for now, but we should do some internal cleaning later to remove duplicate styles.

Reply:
Definitely agree! Once we have the site up and running, I'll create an asana ticket to clean up the codebase a little.